src/main/kotlin/com/cognifide/gradle/aem/deploy/tasks/AbstractTask.kt

```diff
@@ -115,19 +122,19 @@ abstract class AbstractTask : DefaultTask(), AemTask {
             val response = InstallResponse(json)
 
             when (response.status) {
-                InstallResponse.Status.SUCCESS -> if (response.errors.isEmpty()) {
+                AbstractHtmlResponse.Status.SUCCESS -> if (response.errors.isEmpty()) {
                     logger.info("Package successfully installed.")
                 } else {
                     logger.warn("Package installed with errors")
                     response.errors.forEach { logger.error(it) }
                     throw DeployException("Installation completed with errors!")
                 }
-                InstallResponse.Status.SUCCESS_WITH_ERRORS -> {
+                AbstractHtmlResponse.Status.SUCCESS_WITH_ERRORS -> {
                     logger.error("Package installed with errors.")
                     response.errors.forEach { logger.error(it) }
                     throw DeployException("Installation completed with errors!")
                 }
-                InstallResponse.Status.FAIL -> {
+                AbstractHtmlResponse.Status.FAIL -> {
```

Comment:
agreed to leave as it is, with AbstractHtmlResponse name changed to HtmlResponse